benchmark.sh

```diff
@@ -7,35 +7,43 @@ DURATION=120
 [ "$(uname -s)" == "Darwin" ] && SED=gsed
 [ "$(uname -s)" == "Darwin" ] && HEAD=ghead
 
+BENCHMARK_NAME="$1"
+if [ -f "${BENCHMARK_NAME}.yml" ]; then
+  DOCKER_COMPOSE_YML="${BENCHMARK_NAME}.yml"
```

Comment:
Is this condition needed? All docker-compose.yml files seems to be moved into each directory.

Reply:
Oh, sorry. I should have removed this.
I tried keeping backward compatibility first. But I decided that the backward compatibility is needless.

I'll remove this.